.github/workflows/sync-mainnet.yml

```diff
@@ -13,20 +14,14 @@ jobs:
       AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
       AWS_SECRET_KEY: ${{ secrets.AWS_SECRET_KEY }}
       GITHUB_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+      PSQL_HOST: ${{ secrets.PSQL_HOST }}
+      PSQL_PORT: ${{ secrets.PSQL_PORT }}
+      PSQL_USER: ${{ secrets.PSQL_USER }}
+      PGPASSWORD: ${{ secrets.PSQL_PASSWORD }}
```

Comment:
The naming is weird.

Reply:
"PostgreSQL...", but  abbr. to psql, also be ok, and "psql" is the postgresql command line.
but i guess, DB_HOST, DB_PORT, instead of "PSQL_HOST", will be more general

Reply:
done